### PR TITLE
Square text

### LINKED ISSUE #228: Verify square emails
Will need to implement a verification that the square payment is valid and was successful before sending email confirmation to customers using square

### PR DESCRIPTION
**Pull request by amurph24**
close #228 , #206 
PR implements changing of misc db, this only includes price at the moment, and the price is not factored into firewood purchases.
also implements texting and emailing for square purchases

### CODE REVIEW

**Comment by amurph24** [comment]
close #317

**Review by zachpren0 — APPROVED** [review]
Sweeeeeet

File: MERN Scripts/backend/models/models.js
Comment (SpeiserK) [inline MERN Scripts/backend/models/models.js:1]: Good idea to put price in DB

File: MERN Scripts/backend/routes/OrderRoute.js
Comment (SpeiserK) [inline MERN Scripts/backend/routes/OrderRoute.js:1]: Nice work getting texts sent automatically with square order

**Review by SpeiserK — APPROVED** [review]
Nice work!

**Review by camwlsn — APPROVED** [review]
Nice improvements

**Pull request merged**